VideoLocker/src/test/java/org/edx/mobile/test/UtilTests.java

```diff
@@ -32,4 +37,10 @@ public void testHostAndUrls() throws Exception {
         assertFalse(BrowserUtil.isUrlOfHost("https://fake-domain.com/edx.org/", host));
         assertFalse(BrowserUtil.isUrlOfHost("https://fake-domain.com/xyz/", host));
     }
+
+    @Test
+    public void testSha1Encryption() throws UnsupportedEncodingException, NoSuchAlgorithmException {
```

Comment:
oops, this test shouldn't now be throwing those exceptions. I'll update it when the review passes complete.